applications/nrf_desktop/src/hw_interface/leds.c

```diff
@@ -193,8 +247,26 @@ static bool event_handler(const struct event_header *eh)
 
 		__ASSERT_NO_MSG(event->led_id < CONFIG_DESKTOP_LED_COUNT);
 
+		if (streaming && !event->stream) {
+			leds_state_bck[event->led_id] = event->led_effect;
+			return true;
+		}
+
 		struct led *led = &leds[event->led_id];
 
+		led->stream = event->stream;
+
+		if (led->stream) {
+			LOG_INF("Stream event");
+			handle_incoming_effect(event);
+			if (streaming) {
```

Comment:
maybe use `if ... else` instead of multiple `return` ?

Reply:
done